migrations/users/take-this.js

```diff
@@ -10,33 +11,50 @@ let count = 0;
 async function updateUser (user) {
   count++;
 
-  const set = {};
-  let push;
+  let set;
+
+  let migrationsArray = sortBy(user.migrations, ['processed']);
```

Comment:
is the sortBy necessary with an array now? it's already sorted with the last one added at the end